### PR TITLE
docs: smallish changes+fixes while translating (smoe:docs_misc_mac_4)

### PR DESCRIPTION
**Pull request by smoe**
Just to keep the patch small.

### CODE REVIEW

File: docs/man/man9/sserial.9 [hunk 1]
```diff
@@ -414,7 +413,7 @@ higher maximum update rates.
 
 (float out) .7i70.0.0.analogN (modes 1 and 2 only) Analogue input values. 
 
-(float out) .7i70.0.0.fieldvoltage (mode 2 only) Field voltage monitoring pin. 
+(float out) .7______________________i70.0.0.fieldvoltage (mode 2 only) Field voltage monitoring pin.
```
Comment (SebKuzminsky) [inline docs/man/man9/sserial.9:416]: This looks wrong.

**Review by SebKuzminsky — CHANGES_REQUESTED** [review]
Much of this is just changing line breaks.  Is this useful is some way I don't understand?

**Comment by smoe** [comment]
> Much of this is just changing line breaks. Is this useful is some way I don't understand?

These line breaks are affecting the automated translations by some double blanks that are introduced (or by some other magic that I fail to induce). This is particularly unfortunate when text is indented or when some highlighting by _ or * is in the way. I am introducing what I call "semantic line breaks", which is meant to improve (or at least not disturb) the automated interpretation of the documentation.

**Comment by sebastiankuzminsky** [comment]
> These line breaks are affecting the automated translations by some double blanks that are introduced (or by some other magic that I fail to induce).

Ok, I trust you on this one :-)

Thanks for taking such good care of the docs!

**Pull request merged**